### DISCUSSION TITLE
Equivalent of "docker compose down" for Docker-based Talos

### DISCUSSION THREAD

**Discussion by TravisDart** [siderolabs/talos#6935] [Q&A] [answered]
I'm just getting started with Talos, and I created a Docker-based Talos K8S cluster. Now I want to run some other stuff. How do I shut Talos down? I don't want to "destroy" it, do I?

Is there an equivalent of `minikube stop` or `docker compose down` for Talos?

**Comment by smira** [comment, marked as answer]
There is no equivalent of `stop`, but you can `docker stop` for each Talos container
